msg/MultiArrayLayout.msg

```diff
@@ -0,0 +1,30 @@
+# This is an example of using complex datatypes.
+# It is not recommended to use directly.
+# To use a similar datastruct please define a custom message with appropriate semantic meaning.
```

Comment:
@tfoote `datastruct` -- In other places we talk about data types in a similar, almost equivalent. Consider standardizing the terminology.

Reply:
I specifically chose datastruct here to refer to the structure as opposed to the type as the guidance is to make a new type (indirectly by the new message).